howto/update/upgrade-anbox.md

```diff
@@ -138,6 +146,21 @@ In case a reboot of the machine is required, a status message will be shown. Whe
 
     juju run --wait=1m lxd/0 clear-notification
 
+If the LXD charm is deployed on a machine with a NVIDIA GPU installed, the NVIDIA drivers are held from being upgraded by default in case of downtime for all running containers due to either a manual upgrade or an [unattended-upgrade](https://wiki.debian.org/UnattendedUpgrades). The downside to this is that the machine may miss security updates for the NVIDIA drivers. To manually upgrade the NVIDIA drivers, you need to:
+1. Stop the all running containers
+2. Unhold the NVIDIA drivers to allow an upgrade:
+
+        sudo apt-mark unhold linux-modules-nvidia-* nvidia-* libnvidia-*
```

Comment:
```suggestion
        sudo apt-mark unhold 'linux-modules-nvidia-*' 'nvidia-*' 'libnvidia-*'
```

Reply:
We have a task scheduled in this pulse.
Yes, it would be there.
Let's keep it as is.